test/parse_test.dart

```diff
@@ -83,6 +83,79 @@ void main() {
         var results = parser.parse(['--no-b']);
         expect(results['a'], isFalse);
       });
+
+      test('throws if requested as a multi-option', () {
+        var parser = ArgParser();
+        parser.addFlag('a', defaultsTo: true);
+        var results = parser.parse(['--a']);
+        throwsIllegalArg(() => results.multiOption('a'));
+      });
+    });
+
+    group('flag', () {
+      test('are true if present', () {
```

Comment:
Nit: I would work this so that it reads like a phrase when prepended with the surrounding `group()` description, so "returns true if present". That way if the test fails, you'll see a failure on "flag() returns true if present". Likewise below.

Reply:
sgtm - done